src/Dashboard/Statistics/CustomerStatistic.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Dashboard\Statistics;
+
+use App\Repository\CustomerRepository;
+use Symfony\Component\Templating\EngineInterface;
+
+class CustomerStatistic implements StatisticInterface
+{
+    /** @var CustomerRepository */
+    private $customerRepository;
+
+    /** @var EngineInterface */
+    private $engine;
+
+    public function __construct(CustomerRepository $customerRepository, EngineInterface $engine)
+    {
+        $this->customerRepository = $customerRepository;
+        $this->engine = $engine;
+    }
+
+    public function generate(): string
+    {
+        $amountCustomers = $this->customerRepository->countCustomers();
+
+        return $this->engine->render('backend/dashboard/statistics/_amount_of_customers.html.twig', [
+            'amountOfCustomers' => $amountCustomers,
+        ]);
+    }
+
+    public static function getDefaultPriority(): int
```

Comment:
I prefer to define service priorities in service configuration rather than in php code, but still a pretty neat solution 👍